include/alpaka/kernel/TaskKernelGenericSycl.hpp

```diff
@@ -111,15 +110,6 @@ namespace alpaka::detail
     {
     };
 
-    template<typename TElem, typename TIdx, std::size_t TDim, typename TAccessModes>
-    inline auto require(
-        sycl::handler& cgh,
-        Accessor<SyclAccessor<TElem, DimInt<TDim>::value, TAccessModes>, TElem, TIdx, TDim, TAccessModes> acc,
-        special)
-    {
-        cgh.require(acc.m_accessor);
-    }
-
```

Comment:
You can also remove the other `require` overloads I think. We needed them for SYCL accessors to global memory but we don't have those anymore.

Reply:
yes, and therefore also lines 144-145

Reply:
thx, done!